exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/utils/flight_utils.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>What exactly is the transformation we need to do here?</p><p>Can we user the SQL import statements with Cloudfront urls? And, don't need to download the files to AI Lab.</p><p>If this is not possible, we probably don't need a temporary file, because there exists import from callback in Pyexasol which can do the download and transformation at once</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/306/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
The transformation is some kind of data preprocessing, e.g. changing the date format, converting float (0.00/1.00) to boolean, removing ':' from the string, etc.

SQL import is useless, it can only do a limited number of rows, circa 10,000.

I didn't know about the import_from_callback. Seems like a neat way of doing this. Will change the code.

Reply:
OK, it looks much prettier, but works several times slower.